src/docfx/template/liquid/LiquidTemplate.cs

```diff
@@ -52,7 +52,7 @@ public string Render(string templateName, JObject model)
             // if liquid template not found, return the json
             if (template is null)
             {
-                return JsonUtility.Serialize(model);
+                throw Errors.System.LiquidNotFound(mimeType).ToException(isError: false);
```

Comment:
The liquid name is decided by `layout` property, should `layout` be reported too?

Reply:
I thought about that too, but the `layout` is generated by the template script, not set by the user, so it may bring confusion.

Reply:
I thought about that too, but the layout is generated by the template script, not set by the user, so it may bring confusion.

---
In reply to: [414273833](https://github.com/dotnet/docfx/pull/5820#discussion_r414273833) [](ancestors = 414273833)

Reply:
I agree with Jiayin.

---
In reply to: [415189929](https://github.com/dotnet/docfx/pull/5820#discussion_r415189929) [](ancestors = 415189929,414273833)